src/App.js

```diff
@@ -2,15 +2,50 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(prevState => ({ count: prevState.count + 1 }));
+  };
 
-  add100 = () => {};
+  add100 = () => {
+    this.setState(prevState => ({ count: prevState.count + 100 }));
+  };
 
-  increase = () => {};
+  increase = () => {
+    this.addOne();
+    if (this.state.count % 5 === 0) {
```

Comment:
```suggestion
    if (this.state.count % 5 === 0 && this.state.count !== 0) {
```
Тут можно еще добавить такую проверку. Технически ноль то делится на 5, но всё же)

Reply:
Good point